src/include/kernel/ext_manifest.h

```diff
@@ -44,7 +44,9 @@
 #endif
 
 /* In ASCII `XMan` */
-#define EXT_MAN_MAGIC_NUMBER	0x6e614d58
+#define EXT_MAN_MAGIC_NUMBER_SOF	0x6e614d58
+/* In ASCII `$AE1` */
+#define EXT_MAN_MAGIC_NUMBER_CAVS	0x31454124
```

Comment:
This would be better if we just versioned with a number i.e.
```c
#define EXT_MAN_MAGIC_SOF1 0x6e614d58
#define EXT_MAN_MAGIC_SOF2 0x31454124
```
this allows us 3,4 etc in the future and also removes the ```NUMBER``` part which is not needed.